package.json

```diff
@@ -40,7 +40,7 @@
     "cheerio": "0.20.0",
     "commander": "2.9.0",
     "cross-spawn-async": "^2.1.8",
-    "cssom": "0.3.1",
+    "mensch": "brettstimmerman/mensch#v0.3.2",
```

Comment:
Yes, it works here and in the integrations tests.
I guess mensch 0.3.2 is not available in https://www.npmjs.com/package/mensch
mensch 0.3.2 includes a lot of fixes I did to mensch for mosaico that are needed here, too.